frontend/src/model/beta/overview/ModelCardRevisionListDisplay.tsx

```diff
@@ -0,0 +1,31 @@
+import { TableBody, TableCell, TableRow } from '@mui/material'
+import { ThemeProvider, useTheme } from '@mui/material/styles'
+import { useRouter } from 'next/router'
+
+import { ModelCardRevisionInterface } from '../../../../types/v2/types'
+import { formatDateString } from '../../../../utils/dateUtils'
+
+type revisionProp = {
+  modelCard: ModelCardRevisionInterface
+}
+export default function ModelCardRevisionListDisplay({ modelCard }: revisionProp) {
+  const router = useRouter()
+  const { modelId, modelCardVersion }: { modelId?: string; modelCardVersion?: number } = router.query
+  const theme = useTheme()
+
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
We don't need to use this provider as it's already declared at the top level of this application.